table/data_block_hash_index.cc

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2011-present, Facebook, Inc. All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+#include <string>
+#include <vector>
+
+#include "rocksdb/slice.h"
+#include "table/data_block_hash_index.h"
+#include "util/coding.h"
+#include "util/hash.h"
+
+namespace rocksdb {
+
+const uint32_t kSeed = 2018;
+const uint32_t kSeed_tag = 214; /* second hash seed */
```

Comment:
How did you come up with these seeds? Should we use some prime numbers?

Reply:
The Hash() I used (util/hash.cc) is not based on prime, instead, it uses bitwise XOR and rotation to calculate the hash value. 
https://github.com/facebook/rocksdb/blob/1857576e031f1da57dfe153f3ac666aa13ee4a9c/util/hash.cc#L17-L31
It does not have to be prime. Examples: 
https://github.com/facebook/rocksdb/blob/1857576e031f1da57dfe153f3ac666aa13ee4a9c/cache/sharded_cache.h#L85
https://github.com/facebook/rocksdb/blob/1857576e031f1da57dfe153f3ac666aa13ee4a9c/util/hash.h#L27